### PR TITLE
Update Marketplace Content section with information on Studio

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**
@Luyao-Zhang-1 , could you please review this PR?

### CODE REVIEW

File: content/en/docs/developerportal/control-center/_index.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ The **Marketplace Content** page offers a dashboard where you can manage the vis
 
 {{< figure src="/attachments/developerportal/control-center/marketplace-content.png" >}}
 
-The list shows all the Marketplace components for which you have the curation options. This list includes all the public Marketplace components that can be used in Studio and all the private Marketplace components of your company. You can configure whether any of these components is visible in Studio to your company's users.
+The list shows all the Marketplace components for which you have the curation options. This list includes all the public Marketplace components that can be used in Studio and all the private Marketplace components of your company. You can configure whether any of these components is visible in Studio to your company's users (whether widgets appear in the **Toolbox** in Studio).
```
Comment (Luyao-Zhang-1) [inline content/en/docs/developerportal/control-center/_index.md]: Could we make this sentence more generic? Then we don't need to update the text after the scope expands to other types of components as well. Maybe we could say:

(whether components appear in the **Toolbox** in Studio).

Comment (Luyao-Zhang-1) [reply content/en/docs/developerportal/control-center/_index.md]: Below under the **Content Type**, there is a note to tell users that the components are only limited to widgets now.

Comment (MariaShaposhnikova) [reply content/en/docs/developerportal/control-center/_index.md]: @Luyao-Zhang-1 , I changed the term to "components", but I also added one sentence at the very beginning to inform users that currently by "components" we mean "widgets".

Comment (Luyao-Zhang-1) [reply content/en/docs/developerportal/control-center/_index.md]: @MariaShaposhnikova  Thanks. It looks all good to me.

**Pull request merged**